CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.4] - 2020-07-03
+
+### Changed
```

Comment:
I'd call it "Fixed". Do you want to release a new major because of those changes?

Reply:
this has been superseded because i'm an idiot and can't read the docs

Reply:
Ez. I need to come up with some validation for that to annoy people more efficiently :D